test/fixtures/dynamic-argument/expected.7.es2015.js

```diff
@@ -0,0 +1,7 @@
+var MODULE = Object('test-module');
+Promise.resolve("".concat(MODULE)).then(function (s) {
```

Comment:
Note: I need different output files for Babel 7 and 6 because, for example, template literals changed.